exercises/G-numbers/exercise.js

```diff
@@ -1 +1,8 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var numberOfStudents = 22;
+var numberOfMentors = 15;
+
+sum = numberOfStudents + numberOfMentors;
```

Comment:
Hey, love the efficiency of using sum instead of creating another var.

Reply:
Thanks Sino.